Automattic/wp-calypso
# Website content UI shell and REDUX implementation

#### Features completed

* Added the basic UI shell for the website content page
* Integrated UI shell with redux for viewing content and persistence
* Validation and errors of content text fields
* Initial tests for state manipulation

![calypso localhost_3000_start_site-content-collection_website-content_siteSlug=asdas968234344 wordpress com (3)](https://user-images.githubusercontent.com/3422709/151509998-b77e6b55-88e8-4382-96ee-4e7612aab296.png)

![calypso localhost_3000_start_site-content-collection_website-content_siteSlug=asdas968234344 wordpress com (2)](https://user-images.githubusercontent.com/3422709/151510006-a021d8c2-922c-4f9e-9aca-a28d0a72bb04.png)

#### What's not implemented
* Submit step and data submission of the step and to the backend
* Reset image URL
* Data reset once form is completed is not fixed
* Data handling when switching between multiple sites


#### Testing instructions
* Go to `/start/site-content-collection/website-content?siteSlug=<some-site>`
* Make sure validation works
* Make sure UI is correctly implemented
* Browser persistence of data should work



Related to #59895
